server/webapp/WEB-INF/rails.new/webpack/views/shared/info_callout.js.msx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2018 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const m = require('mithril');
+const f = require('helpers/form_helper');
+
+class InfoCalloutWidget {
```

Comment:
I noticed that this is already defined in `form_helper`, and that we're just wrapping it in a `row`. I'll probably just move this to `form_helper` and name it `calloutRow` or something